pkg/server/gitproviders/service.go

```diff
@@ -81,14 +81,17 @@ func (s *GitProviderService) GetLastCommitSha(repo *gitprovider.GitRepository) (
 	}
 
 	for _, p := range gitProviders {
-		if p.Id == "aws-codecommit" && strings.Contains(repo.Url, ".amazonaws.com/") {
+
+		var isAwsUrl bool = strings.Contains(repo.Url, ".amazonaws.com/") || strings.Contains(repo.Url, ".console.aws.amazon.com/")
```

Comment:
`isAwsUrl := ....`

Reply:
That's okay. Just saying that `isAwsUrl := ` is better than `var isAwsUrl bool = `